src/components/home/Home.tsx

```diff
@@ -5,7 +5,6 @@ import Footer from "../footer/Footer";
 import ErrorBoundary from "../errorBoundary/ErrorBoundary";
 import Logo from "../ui/Logo/Logo";
 import {SuspenseWrapper} from "../../styles/general";
-import {DialogProvider} from "../../context/modalMovieContext";
 
 // this error disappear if I export components as React.FC, bit then new errors appear
 const FilmPopup =  React.lazy(() => import(`../filmPopups/FilmForm`));
```

Comment:
Considering you lazy load certain components, you could've benefited from [Redux code-splitting](https://redux.js.org/recipes/code-splitting) just alike. You may find the example of that in the [repo](https://git.epam.com/yevhen_horbunkov/heimdall/-/blob/develop/src/client/store/storeConfig.ts) I've shared previously.